### PR TITLE
Import material (product) data from SAP

### PR DESCRIPTION
**Pull request by mi-ps**
refs: https://github.com/metasfresh/metasfresh/issues/13745

### CODE REVIEW

File: misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-sap/src/main/java/de/metas/camel/externalsystems/sap/api/model/product/ProductRow.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	@DataField(pos = 1)
+	private String materialCode;
+
+	@DataField(pos = 3)
+	private String sectionCode;
+
+	@DataField(pos = 4)
+	private String name;
+
+	//FIXME: pending mappings
```
Comment (metas-ts) [inline misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-sap/src/main/java/de/metas/camel/externalsystems/sap/api/model/product/ProductRow.java:42]: is this still current? seems to me that you do use the value

**Comment by metas-ts** [comment]
also, please add the metasfresh-Issue-URL to this PR's description

File: misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-sap/src/main/java/de/metas/camel/externalsystems/sap/api/model/product/ProductRow.java (hunk 1 above)
Comment (cp-ps) [reply misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-sap/src/main/java/de/metas/camel/externalsystems/sap/api/model/product/ProductRow.java:42]: yep, this is still a thing as according to specs the UOM is in some custom format => so we might need custom mappings

**Review by metas-ts — CHANGES_REQUESTED** [review]
lgtm so far, but please check my remarks regarding Proposed TRLs

**Pull request merged**